src/main/java/org/apache/commons/lang3/reflect/MethodUtils.java

```diff
@@ -92,6 +92,29 @@ public static Object invokeMethod(final Object object, final String methodName)
             IllegalAccessException, InvocationTargetException {
         return invokeMethod(object, methodName, ArrayUtils.EMPTY_OBJECT_ARRAY, null);
     }
+    
+    /**
+     * <p>Invokes a named method without parameters.</p>
+     *
+     * <p>This is a convenient wrapper for
+     * {@link #invokeMethod(Object object,String methodName, Object[] args, Class[] parameterTypes)}.
```

Comment:
I guess this should be ``invokeMethod(final Object object, final boolean forceAccess, final String methodName, Object[] args, Class[] parameterTypes)`?